tensorflow_serving/servables/tensorflow/session_bundle_config.proto

```diff
@@ -86,4 +86,7 @@ message BatchingParameters {
   //  - The entries must be in increasing order.
   //  - The final entry must equal 'max_batch_size'.
   repeated int64 allowed_batch_sizes = 6;
+
+  // Whether to pad variable-length inputs when a batch is formed.
+  google.protobuf.BoolValue pad_variable_length_inputs = 7;
```

Comment:
Can we just use "bool" instead of BoolValue? We don't need three-valued logic here, i.e. just defaulting to "false" if it's omitted is fine right?